scaffolder-templates/github-workflows/basic-workflow/docs/GithubBasicDoc.md

```diff
@@ -0,0 +1,112 @@
+# Creating a Basic Serverless Workflow via a Software Template with GitHub
+
+## Overview
+This template allows the user to generate a basic workflow project from the user's input.
+The workflow will be generated corresponding to the workflow type (assessment, infrastructure) the user selects.
+
+### basic-workflow as assessment
+An assessment workflow project is generated from the Github Basic Workflow or the Gitlab Basic Workflow templates. This is a simple workflow that evaluates the user's input as string by comparing it to the word `default` to print out the default workflow options or an empty workflow options list.
+Unlike the infrastructure workflow, the output schema of the assessment workflow is crucial to get the workflow options rendered in the UI. In this basic assessment, there is no data output schema validation but the generated workflow provides a sample object for reference which is made of six mandatory fields (currentVersion, upgradeOptions, migrateOptions, newOptions, continuationOptions, otherOptions). See below an example:
```

Comment:
nitpick: add a newline before this paragraph to ease the reading

Reply:
> This is a simple workflow that evaluates the user's input as string by comparing it to the word `default`

Again, where did you see that?  https://github.com/rhdhorchestrator/workflow-software-templates/blob/main/scaffolder-templates/skeletons/basic-workflow-skeleton/src/main/resources/assessment-template.sw.yaml#L43C1-L50C42
```
states:
  - name: StartState
    type: switch
    dataConditions:
      - condition: ${ .sampleText | length != 0 }
        transition: GetSampleWorkflowOptions
    defaultCondition:
      transition: GetEmptyWorkflowOptions
```

Reply:
> Unlike the infrastructure workflow

If you mention the infra workflow, it should be listed before the assessment one

Reply:
> Github Basic Workflow or the Gitlab Basic Workflow

Is it possible to add link to the folders? Should the user be aware of this technicality? Is the target user an admin or an user of the template?